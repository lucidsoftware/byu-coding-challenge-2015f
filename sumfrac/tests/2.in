30
534/5
782/1
92/2
118/1
809/9
749/1
900/6
86/4
543/1
350/3
411/3
537/7
782/3
856/10
456/10
53/5
42/5
445/6
880/4
16/5
357/5
390/2
549/2
656/5
546/4
379/5
954/4
572/7
354/2
745/2