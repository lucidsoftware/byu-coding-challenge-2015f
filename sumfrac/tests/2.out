1700659/315
